dask/bytes/core.py

```diff
@@ -0,0 +1,156 @@
+import io
+import sys
+
+from toolz import merge
+
+from .compression import seekable_files, files as cfiles
+from .utils import SeekableFile
+from ..delayed import delayed
+from ..utils import system_encoding
+
+delayed = delayed(pure=True)
+
+_read_bytes = dict()
+_open_files = dict()
+_open_text_files = dict()
+
+
+def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
+                     sample=True, compression=None, **kwargs):
+    """ Convert path to a list of delayed values
+
+    The path may be a filename like ``'2015-01-01.csv'`` or a globstring
+    like ``'2015-*-*.csv'``.
+
+    The path may be preceeded by a protocol, like ``s3://`` or ``hdfs://`` and,
+    if those libraries are installed, the futures will point to those locations
```

Comment:
"those locations instead" is confusing; I would write "The path may be preceded by a protocol, like ``s3://`` or ``hdfs://``, if those libraries are installed.